src/charts.js

```diff
@@ -186,9 +186,38 @@
     };
 
     // Merge defaults and options into settings
-    var settings = $.extend( true, {}, defaultSettings, options );
-    $.each( Object.keys( settings.measures ), function( index, measure ) {
-      settings.measures[ measure ] = $.extend( true, {}, genericMeasureDefaults, settings.measures[ measure ] );
+    function merge_objects(obj1, obj2){
```

Comment:
We plan to pull all of the changes into a feature branch so that we can add some tests, and we may integrate some additional features in this merge function that will facilitate unsetting default configuration keys.